docs/guides/admin/docs/workflowoperationhandlers/publish-engage-woh.md

```diff
@@ -28,9 +28,9 @@ Parameter Table
 |streaming-source-tags      |Specifies which media should be published to the streaming server                            |
 |streaming-target-tags      |Add tags (comma separated) to published media                                                |
 |streaming-target-subflavors|Subflavor to use for distributed material                                                    |
-|merge-force-flavors        |Flavors of elements for which an update is enforced when merging catalogs.                  |
+|merge-force-flavors        |Flavors of elements for which an update is enforced when merging catalogs.                   |
 |                           |Defaults to `dublincore/*,security/*`.
-
+|list-elements              |Include all media package elements in the publication. Defaults to `true`.                   |
```

Comment:
The consensus during the dev meeting seems to be to always enable listing the published elements. So we don't need this option.

Reply:
Sorry, I lost track of this PR. Removed the option to disable listing the published elements. 

It seems that the `retract-partial` WOH is currently not able to find the added elements. I need to check that...

Reply:
Actually it was a user error on my side...tested partialy retracting some published elements and everything seems to work as expected.